recipes/feather-format/meta.yaml

```diff
@@ -0,0 +1,36 @@
+package:
+  name: feather-format
+  version: "0.2.0"
+
+source:
+  fn: feather-format-0.2.0.tar.gz
+  url: https://pypi.python.org/packages/06/bb/072a00246f235df6a902dbaf4cb10d687ea5b3a199f786dd99c23cd08cf8/feather-format-0.2.0.tar.gz
```

Comment:
Sorry, probably should have included that in the PR. I was more focused on getting it building quickly.